CMakeLists.txt

```diff
@@ -612,9 +616,6 @@ endif()
 #
 find_program(LUACHECK_PRG luacheck)
 find_program(STYLUA_PRG stylua)
-find_program(FLAKE8_PRG flake8)
-find_program(UNCRUSTIFY_PRG uncrustify)
-find_program(SHELLCHECK_PRG shellcheck)
```

Comment:
I don't understand why these were moved. Isn't the FOO_PRG detection unrelated to the idea of creating "glob targets"?